Library/Homebrew/test/rubocops/lines_cop_spec.rb

```diff
@@ -419,7 +419,22 @@ class Foo < Formula
       end
     end
 
-    it "with ENV.universal_binary" do
+    it "with ENV.universal_binary exempted formula" do
```

Comment:
Can you make this into a proper sentence? Thanks. 😉

Reply:
I was only matching the one above it. So, inconsistency or proper sentences?

Reply:
Ideally proper sentences for all, or `specify` instead of `it`.

Reply:
The full fragment is: `' When auditing formula with ENV.universal_binary exempted formula'` any suggestions?

Reply:
(I am referring to the line https://github.com/Homebrew/brew/pull/3291/files#diff-9349b511bf2b9967d30735fbb26db7b7R383)